lib/symboldatabase.cpp

```diff
@@ -2555,6 +2555,8 @@ Function::Function(const Token *tok,
     if (::isOperator(tokenDef)) {
         isOperator(true);
 
+        isExplicit(tokenDef->strAt(-1) == "explicit" || tokenDef->strAt(-2) == "explicit");
```

Comment:
could we search backwards for a explicit keyword instead.. in case there will be more additional keywords.

    explicit foo bar operator bool...

I don't know if there can be some c++ attribute between.. but it feels like it would be nice to handle this also.

    explicit [[foo]]  operator bool...